fix: make `<build>` route exclusion rule only exclude static assets (#13411)
sveltejs/kit: .changeset/chilled-timers-give.md

---
'@sveltejs/adapter-cloudflare': patch
---

> fix: make `<build>` route exclusion rule only exclude static assets

Reviewer: ```suggestion
fix: exclude the dynamic route `/_app/env.js` from the `<build>` route exclusion rule
```
